python/py-nibabel/Portfile

```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
 PortGroup               python 1.0
 
 name                    py-nibabel
-version                 2.3.3
+version                 2.4.0
 revision                0
 platforms               darwin
 license                 MIT
```

Comment:
it appears that nothing gets compiled / architecture-dependent files are installed. If that observation is correct, then it would be good to add ```supported_archs     noarch```